example_pipeline_custom.sh

```diff
@@ -5,14 +5,22 @@
 # Prostate Cancer. The data is used as a custom input.
 set -o errexit
 
-# Define filenames
-candidate_snp_file='custom_example.csv'
-candidate_snp_location_file='results/custom_example_location.tsv'
-candidate_snp_tad_file='results/custom_example_tad_results.tsv'
-nearest_gene_file='results/custom_example_tad_results_nearest_gene.tsv'
+# Set constants
 trait='custom'
-evidence_file='results/custom_gene_evidence.csv'
-pathway_p_values_file='gestalt/custom_gestalt.tsv'
+custom_dir='results/custom_example/'
+geneset='geneontology_Biological_Process'
+pathway_name='skeletal_system_development'
+
+# Set input file names
+candidate_snp_file='custom_example.csv'
+
+# Set output file names
+candidate_snp_location_file=$custom_dir'custom_example_location.tsv'
```

Comment:
usually the custom is to include a `/` separator here. If the path ends up without the separator = BIG PROBLEM. If it ends up with `//` as a separator = NOT PROBLEM.